mesos-docker/run/run.sh

```diff
@@ -89,9 +90,6 @@ function generate_application_conf_file {
   sed -i -- "s@replace_with_docker_host_ip@$host_ip@g" "$target_location/mit-application.conf"
   sed -i -- "s@replace_with_spark_executor_ui@$spark_tgz_file@g" "$target_location/mit-application.conf"
   
-  #remove any temp file generated
-  rm "$target_location/mit-application.conf--"
```

Comment:
Yes, it is attempting to remove a file which is never created.
I think it was use as a temporary file for the sed operations during development.